web/src/invite.js

```diff
@@ -351,6 +352,15 @@ function open_invite_user_modal(e) {
         $("#invite_select_default_streams").on("change", () => {
             set_streams_to_join_list_visibility();
         });
+
+        if (!settings_data.user_can_invite_others_to_realm()) {
+            $("#email_invite_radio").prop("disabled", true);
+            $("#email_invite_radio_container").addClass("control-label-disabled");
+            $("#email_invite_radio_container").addClass("disabled_setting_tooltip");
+
+            $("#generate_multiuse_invite_radio").prop("checked", true);
+            $("#generate_multiuse_invite_radio").trigger("change");
```

Comment:
I added this code after adding all event listener so that we can avoid some duplicate code by using `$("#generate_multiuse_invite_radio").trigger("change")`